lib/rubocop/ast/node.rb

```diff
@@ -453,6 +453,10 @@ def binary_operation?
           source_range.begin_pos != loc.selector.begin_pos
       end
 
+      def parenthesized_call?
```

Comment:
Won't `SendNode` be a better home for this?

Reply:
Check out the description:

Util#parenthesized_call? is only invoked in the 2 files seen in the diff below. While the method did have an argument named 'send', it is used with other types of nodes as well. However, in all places #parenthesized_call? is used, there are guard clauses that guarantee it is only passed a node and not anything else. Since the method operates solely on Node, it is Node's and not a Cop's responsible.

---

I moved it to `SendNode` originally, but the test suite calls it on other types of nodes. Right [here](https://github.com/bbatsov/rubocop/pull/5354/commits/3bcfcf558a35c2305de6b47260aabd1beeab87bc#diff-2b41a1cdba20584c980b55084250637cR75) specifically.